br/pkg/conn/conn_test.go

```diff
@@ -406,6 +406,8 @@ func TestGetMergeRegionSizeAndCount(t *testing.T) {
 	}
 
 	ctx := context.Background()
+	failpoint.Enable("github.com/pingcap/tidb/br/pkg/conn/stop-retry-on-get-config-from-tikv", "return(true)")
```

Comment:
It should check whether to return the no error. it will return an error if failpoint is deleted.